modules/ROOT/pages/_partials/cli-auth.adoc

```diff
@@ -0,0 +1,210 @@
+----
+$ api-catalog conf username myuser
+$ api-catalog conf password mypwd
+----
+
+The commands store the credential information in the `config.json` file in the following locations:
+
+* MacOS: `~/Library/Preferences/@mulesoft/api-catalog-cli-nodejs/config.json`
+
+* Windows: `%LOCALAPPDATA%\mycli`
+
+* Unix: `~/.config/mycli`
+
+*Examples:*
+
+* Set a value for a bearer token: 
++
+----
+$ api-catalog conf bearer b71cab22-e5cc-43c4-90ed-5a1124b6ae1f
+----
++
+* Set values for username and password: 
++
+----
+$ api-catalog conf username myuser
+$ api-catalog conf password mypwd
+----
++
+* Set values for a specific host, organization, and environment:
++
+----
+$ api-catalog conf host myco.anypoint.mulesoft.com
+$ api-catalog conf session.selectedEnvironment.id ec7d16cb-43fd-4796-8b4f-5cff7294fcf9
+$ api-catalog conf session.selectedOrganization.id 8a840abc-e63a-4f8d-87ad-24052eda3018
+----
++
+* Set values for client ID and client secret:
++
+----
+$ api-catalog conf client_id myClientID
+$ api-catalog conf client_secret myCLientSecret
+----
+
+Example `config.json` contents after running the above commands:
+
+----
+ {
+	"bearer": "b71cab22-e5cc-43c4-90ed-5a1124b6ae1f",
+    "username": "myuser",
+	"password": "mypwd",
+    "host": "myco.anypoint.mulesoft.com",
+	"session": {
+		"selectedEnvironment": {
+			"id": "ec7d16cb-43fd-4796-8b4f-5cff7294fcf9"
+		},
+		"selectedOrganization": {
+			"id": "8a840abc-e63a-4f8d-87ad-24052eda3018"
+		}
+	},
+	"client_id": "1787c36ab544466698e380131040faad",
+	"client_secret": "2dA616efca1E4181A2A77BBF046868BE"
+ }
+----
+
+// end::conffile[]
+
+// tag::cmdline[]
+
+[[command-line-options]]
+== Command Line Options
+
+You can authenticate to Anypoint Platform through command line by running the `api-catalog` command with a combination of the following options:
+
+----
+--username
+--password
+--client_id
+--client_secret
+--bearer
+--organization
+--environment
+----
+
+*Examples:*
+
+* Authenticate on the command line using username and password
++
+----
+$ api-catalog --username myUserName --password myPassword --organization myOrgId account:environment:list
+----
+
+* Authenticate on the command line  using client ID and client secret
++
+----
+$ api-catalog --client_id myClientID myClientSecret --organization myOrgId account:environment:list
+----
+
+* Authenticate on the command line  using a bearer token
++
+You can generate a bearer token in Anypoint Platform and pass it to API Catalog CLI directly. When you provide a bearer token, API Catalog CLI ignores any provided user or client credentials.
++
+----
+$ api-catalog --bearer myBearerToken --organization myOrgId account:environment:list
+----
++
+Your Anypoint Platform session expires when the bearer token expires.
++
+For information about generating a bearer token, see https://help.mulesoft.com/s/article/How-to-generate-your-Authorization-Bearer-token-for-Anypoint-Platform[How to Generate Your Authorization Bearer Token for Anypoint Platform].
```

Comment:
```suggestion
For information about generating a bearer token, see https://help.mulesoft.com/s/article/How-to-generate-your-Authorization-Bearer-token-for-Anypoint-Platform[How to Generate Your Authorization Bearer Token for Anypoint Platform^].
```

Reply:
Yep, thanks. Just forgot it on this one. :)